HPXMLtoOpenStudio/resources/output.rb

```diff
@@ -993,6 +993,94 @@ def self.get_total_hvac_capacities(hpxml_bldg)
     return htg_cap, clg_cap, hp_backup_cap
   end
 
+  # Calculates total panel loads (across all panel loads for a given panel load type) for a given HPXML Building.
+  #
+  # @param hpxml_bldg [HPXML::Building] HPXML Building object representing an individual dwelling unit
+  # @return [Array<Double * 15>] Total panel load for each panel load type (W)
+  def self.get_total_panel_loads(hpxml_bldg)
```

Comment:
We should revisit that. Service entrance for building has a different set of demand factors that needs to be applied for load coincidence, in addition to unit_multiplier. Let's discuss.